Щукин Георгий М3336 ДЗ1

memesHunter opened the pull request:
Из всех пунктов задания нету только центрирования активной в галереи-карусели фотографии. Но очень хотелось запушить до дедлайна.

Commit:
Initial solution

Yuliana-coder commented on about-cats/src/index.css:
```diff
@@ -1 +1,273 @@
+  /* Gallery cards container
+  ============================ */
+
+  div.gallery-block {
+    text-align: center;
+    vertical-align: top;
+  }
+  
+  /* Single gallery card
+  ============================ */
+  div.gallery {
+    border: 1px solid #aaa;
+    display: inline-block;
```
почему все селекторы через div?

Yuliana-coder replied:
и вообще во всех селекторах есть теги

Yuliana-coder commented on about-cats/src/index.html:
```diff
@@ -4,8 +4,191 @@
-    <!-- TODO -->
+
+    <div class="header-nav">
+      <p>Мурмаркет</p>  
```
по макету первая буква должна быть красной

Yuliana-coder commented on about-cats/src/index.html:
```diff
@@ -4,8 +4,191 @@
+
+
+      <div class="gallery-block">
+        <div class="gallery">
+          <img src="cats/cat1.jpg" alt="cat1">
+          <div class="desc">Музыкальный кот</div>
+        </div>
```
сильно отличается от макета, можно добавить хотя бы небольшое описание в карточке

Yuliana-coder commented on about-cats/src/index.html:
```diff
@@ -4,8 +4,191 @@
+        <p class="contacts">
+          Контакты: <br>
+          <a href="mailto: cats@babuli.ru">cats@babuli.ru</a><br>
+          <a href="mailto: vnuchek@babuli.ru">vnuchek@babuli.ru</a><br>
```
нет иконок у почт

Yuliana-coder commented on about-cats/src/index.html:
```diff
@@ -4,8 +4,191 @@
+          <div class="desc">Музыкальный кот</div>
+        </div>
+      
+        <div class="gallery">
+          <img src="cats/cat2.jpg" alt="cat2">
+          <div class="desc">Потревоженный кот</div>
+        </div>
```
в макете заголовок справа наверху

Yuliana-coder commented:
@memesHunter привет, я глянула, посмотри, пожалуйста, комменты

trixartem commented:
Жду исправления комментариев

Commit:
Убрал теги из селекторов по классам;
Первая буква названия красная;
Добавил описание для карточек;
Иконки для почт;
Имя в модальном окне справа.

trixartem commented:
✅ Принято
